x/kavamint/genesis.go

```diff
@@ -0,0 +1,29 @@
+package kavamint
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/kava-labs/kava/x/kavamint/keeper"
+	"github.com/kava-labs/kava/x/kavamint/types"
+)
+
+// InitGenesis new mint genesis
+func InitGenesis(ctx sdk.Context, keeper keeper.Keeper, ak types.AccountKeeper, data *types.GenesisState) {
+	keeper.SetParams(ctx, data.Params)
+
+	// only set the previous block time if it's different than default
+	if !data.PreviousBlockTime.Equal(types.DefaultPreviousBlockTime) {
+		keeper.SetPreviousBlockTime(ctx, data.PreviousBlockTime)
+	}
+
+	ak.GetModuleAccount(ctx, types.ModuleName)
```

Comment:
i'll add one as a sanity check 👍 
pretty sure it can never return nil as it creates one if it doesn't exist yet. reality probably has "can never" -> "should never" so a check seems reasonable

https://github.com/Kava-Labs/cosmos-sdk/blob/d01aa5b4a8776f09656c9d64586dfd7f924d69eb/x/auth/keeper/keeper.go#L221-L226